features/bookmarks/bookmark_filter.feature

```diff
@@ -17,3 +17,55 @@ Feature: Filter bookmarks
     Then I should see "2 Bookmarks by recengine"
       And I should see "english work"
       And I should see "german work"
+
+  Scenario: Filtering series bookmarks by word count with restricted works
+    Given I am logged in as "awesome_posterrr"
+      And I post the work "Public" as part of a series "Mixed Visibility"
+      And I post the work "Restricted" as part of a series "Mixed Visibility"
+      And I lock the work "Restricted"
+      And I bookmark the series "Mixed Visibility"
+    When I go to awesome_posterrr's bookmarks page
+    # Verify the starting visible word count
+    Then I should see "1 Bookmark by awesome_posterrr"
+      And I should see "Words: 12"
+    When I fill in "To" with "10" within "#work_words"
+      And I press "Sort and Filter"
+    Then I should see "0 Bookmarks by awesome_posterrr"
+      And I should not see "Mixed Visibility"
+    When I fill in "To" with "20" within "#work_words"
+      And I fill in "From" with "5" within "#work_words"
+      And I press "Sort and Filter"
+    Then I should see "1 Bookmark by awesome_posterrr"
+      And I should see "Mixed Visibility"
+    When I am logged out
+      And I go to awesome_posterrr's bookmarks page
+    Then I should see "1 Bookmark by awesome_posterrr"
+      And I should see "Words: 6"
+    When I fill in "To" with "10" within "#work_words"
+      And I press "Sort and Filter"
+    Then I should see "1 Bookmark by awesome_posterrr"
+      And I should see "Mixed Visibility"
```

Comment:
The last two steps here keep failing, and I can't figure out why. The first part of the test works, and filtering works for localhost :thinking:

Reply:
It was due to a lack of reindexing, I'm pretty sure